simple_ddl_parser/ddl_parser.py

```diff
@@ -238,7 +238,7 @@ def p_id(self, p):
         delimiters_to_end = ["`", '"', "]"]
         p[0] = p[1]
 
-        if self.normalize_names:
+        if self.normalize_names and len(p[0]) > 2:
```

Comment:
this was causing `[]` to be removed, which are needed in statements like `decimal(21)[]`